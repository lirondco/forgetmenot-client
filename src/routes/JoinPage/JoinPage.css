.register_header {
    padding: 4%;
    text-align: center;
    font-family: 'Patrick Hand SC', cursive;
    font-size: 28pt;
}


