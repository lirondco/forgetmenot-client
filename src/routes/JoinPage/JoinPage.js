import React, { Component } from 'react'
import { Section } from '../../components/Utils/Utils'
import Join from '../../components/Join/Join'
import './JoinPage.css'

export default class JoinPage extends Component {
  static defaultProps = {
    history: {
      push: () => {},
    },
  }

  handleRegistrationSuccess = user => {
    const { history } = this.props
    history.push('/')
  }

  render() {
    return (
      <Section className='RegistrationPage'>
        <h2 className="register_header">Register</h2>
        <Join
          onRegistrationSuccess={this.handleRegistrationSuccess}
        />
      </Section>
    )
  }
}