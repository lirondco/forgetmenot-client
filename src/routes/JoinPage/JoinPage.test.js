import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import JoinPage from './JoinPage'

describe(`JoinPage component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<JoinPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})