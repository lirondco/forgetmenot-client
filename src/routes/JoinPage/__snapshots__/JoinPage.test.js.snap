// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JoinPage component renders the complete form 1`] = `
<Section
  className="RegistrationPage"
>
  <h2
    className="register_header"
  >
    Register
  </h2>
  <Join
    onRegistrationSuccess={[Function]}
  />
</Section>
`;
